# Waypoint.msg

Header header
uint16 number
uint16 next_number
geometry_msgs/Vector3 lla
float32 orbit_time
bool fly_through
string description
